src/workerd/server/workerd.capnp

```diff
@@ -651,6 +654,10 @@ struct Worker {
   streamingTails @15 :List(ServiceDesignator);
   # List of streaming tail worker services that should receive tail events for this worker.
   # NOTE: This will be deleted in a future refactor, do not depend on this.
+
+  dockerSocketPath @16 :Text;
```

Comment:
Note for later:

Maybe we want to make this a `DockerConnection` struct instead of a simple address at some point.

There are [multiple ways](https://docs.rs/bollard/latest/bollard/struct.Docker.html#method.connect_network) to connect to docker, we could also have a version, timeout, ...

Maybe add a todo comment for later. Or create a `struct DockerConnection` already with a single `address: Text` field fow now, that might be better for future extensions.

Reply:
Yes, that's really good! I'm changing this as you recommended.